NativeAot/nuget.config

```diff
@@ -9,10 +8,17 @@
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
     <add key="nuget" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
+    <add key="local_packages" value="../Terminal.Gui/local_packages" />
   </packageSources>
   <packageSourceMapping>
     <packageSource key="nuget">
       <package pattern="*" />
     </packageSource>
+	<packageSource key="LocalPackages">
+	  <package pattern="Terminal.Gui" />
+	</packageSource>
```

Comment:
I think this can be removed because only the `local_packages` is used in the `packageSources`.